### PR TITLE
APPDRUSTI-28 Cotact page

### PR DESCRIPTION
**Pull request by BOSSHK-SPEC**

### CODE REVIEW

File: lib/MyProfile.dart [hunk 1]
```diff
@@ -749,24 +764,184 @@ class MyProfileState extends State {
...
+                                decoration: new InputDecoration(
+                                    isDense: true,
+                                    contentPadding: new EdgeInsets.symmetric(
+                                        vertical: 10.0, horizontal: 10.0),
+                                    focusedBorder: OutlineInputBorder(
+                                      borderSide: BorderSide(
+                                          color: Colors.black, width: 1.0),
+                                    ),
+                                    enabledBorder: OutlineInputBorder(
+                                      borderSide: BorderSide(
```
Comment (sahana-drusti) [inline lib/MyProfile.dart]: add email validator similar to login page

File: lib/MyProfile.dart [hunk 2]
```diff
@@ -5,6 +5,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:http/http.dart' as http;
 import 'package:shared_preferences/shared_preferences.dart';
+import 'package:email_validator/email_validator.dart';
```
Comment (sahana-drusti) [inline lib/MyProfile.dart]: remove this dependency

File: lib/MyProfile.dart (hunk 1 above)
Comment (BOSSHK-SPEC) [reply lib/MyProfile.dart]: done

File: lib/MyProfile.dart (hunk 2 above)
Comment (BOSSHK-SPEC) [reply lib/MyProfile.dart]: done

**Review by sahana-drusti — APPROVED** [review]

**Comment by sahana-drusti** [comment]
unwanted changes reflecting

**Pull request closed without merging**